src/components/Reviews.jsx

```diff
@@ -0,0 +1,28 @@
+import { useEffect } from "react";
+import { getReviews } from "../requests";
+
+const Reviews = ({ reviews, setReviews }) => {
+  useEffect(() => {
+    getReviews().then((data) => {
+      setReviews(data);
+    });
+  }, []);
+  return (
+    <section>
```

Comment:
We should give users some visual feedback here for when the page is loading and waiting for the fetch request to complete.